nixos/modules/services/monitoring/watchdogd.nix

```
@@ -0,0 +1,128 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.watchdogd;
+
+  mkPluginOpts = plugin: {
+    enable = mkEnableOption "watchdogd plugin ${plugin}";
+    interval = mkOption {
+      type = types.int;
+      default = 300;
+      description = ''
+        The poll interval. Defaults to 300 seconds.
```

I don't think the defaults need to be included in the descriptions. They will be printed in the manual anyway.

```suggestion
        Amount of seconds between every poll.
```